clarifaipyspark/dataset.py

```diff
@@ -415,11 +415,8 @@ def export_inputs_to_dataframe(self, input_type):
     """
     if input_type not in ('image', 'text'):
       raise UserError('Invalid input type, it should be image or text')
-    
-    search_obj = Search(user_id=self.user_id, app_id=self.app_id, pat=self.pat)
-    search_response = search_obj.query(filters=[{"input_types":["image"]},{"input_dataset_ids":[self.dataset_id]}])
-    inputs=[hit.input for response in search_response for hit in response.hits]
-
+  
+    inputs= list(self.list_inputs(input_type=input_type))
```

Comment:
btw, shouldn't these be filtered by `dataset_id` ?

Reply:
Since we are initialising dataset class in the beginning and calling the methods using `self.list_inputs`, we don't need to specify the `dataset_id` filter here.

Reply:
Gotcha 👍